certified-connectors/StarRez REST v1/apiDefinition.swagger.json

```diff
@@ -0,0 +1,17799 @@
+                  },
+                  "Position": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Position"
+                  },
+                  "Previous_EntryStatusEnum": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Previous Entry Status Enum"
+                  },
+                  "TaxExemptionEnum": {
+                    "type": "string",
+                    "description": "",
+                    "title": "Tax Exemption Enum"
+                  },
+                  "Testing": {
+                    "type": "boolean",
+                    "description": "Testing",
+                    "title": "Testing",
+                    "enum": [
+                      true,
+                      false
+                    ]
+                  }
+                }
+              }
+            }
+          }
+        },
+        "operationId": "Select_Entry",
+        "summary": "Get Entry",
+        "description": "Retrieves one or more records from the Entry table.",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "_pageSize": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Maximum number of records to return.",
+                  "title": "Page Size",
+                  "enum": [
+                    5,
+                    10,
+                    50,
+                    100,
+                    500,
+                    1000
+                  ],
+                  "default": 1000,
+                  "x-ms-visibility": "important"
+                },
+                "_pageIndex": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Index of the first record to return.",
+                  "title": "Page Index",
+                  "x-ms-visibility": "important",
+                  "default": 0
+                },
+                "_top": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Returns the top results up to the specified number of records.",
+                  "title": "Top",
+                  "x-ms-visibility": "important"
+                },
+                "_orderby": {
+                  "type": "string",
+                  "description": "Sort by one or more fields in ascending order. Fields must be separated by commas with no spaces (e.g. field1,field2).",
+                  "title": "Order By",
+                  "x-ms-visibility": "important"
+                },
+                "AddressTypeID": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "",
```

Comment:
@Tony747SREZ please define a proper description. please fix all parameters' empty description in the swagger file.

Reply:
Sorry, probably our misinterpretation. We were thinking that the term parameter meant either a field in the Request Path or a Required field in the Request Body.

However it seems every field needs to have a description. Is this for both the Request and Response? 

Assuming we need to create descriptions for all fields, would there be a problem using descriptions that simply reinforce the existing naming of the field? Eg would it be ok to use "Unique ID of the Address Type"? even though it does not offer any additional meaning that could be gleaned from the name.

Reply:
@Tony747SREZ any string should work. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters).